<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="RAVIG-WIN10" failures="1" tests="6" name="PetTest" time="10.272" errors="0" timestamp="2025-07-20T11:44:12 GMT-07:00">
  <testcase classname="api.test.DDTests" name="testPostUser" time="2.776"/>
  <testcase classname="api.test.DDTests" name="testPostUser" time="1.251"/>
  <testcase classname="api.test.DDTests" name="testPostUser" time="1.259"/>
  <testcase classname="api.test.DDTests" name="testDeleteUserByUserName" time="1.288"/>
  <testcase classname="api.test.DDTests" name="testDeleteUserByUserName" time="1.238">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.DDTests.testDeleteUserByUserName(DDTests.java:61)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByUserName -->
  <testcase classname="api.test.DDTests" name="testDeleteUserByUserName" time="1.297"/>
</testsuite> <!-- PetTest -->
